include/Einleitung.tex

```diff
@@ -1,12 +1,12 @@
+In der Kooperativen Speicherbibliothek werden Exemplare der dem Verein zugehörigen Bibliotheken eingelagert. Das heisst, das Exemplar der Bibliothek wird gereinigt und inventarisiert eingelagert. Sollte ein Bibliotheksbenutzer ein Exemplar dieses Werks anfordern, wird das Exemplar per Kurier an die jeweilige Bibliothek verschickt. Journale oder Magazine werden vorzugsweise eingescannt und in digitaler Form an den Benutzer übergeben, sofern dies so vom Benutzer bestellt wurde. Sowohl die beteiligten Bibliotheken, wie auch die Bibliotheksbenutzer sind daher an einer zeitgerechten Lieferung interessiert. Dies bedeutet für die Speicherbibliothek, dass alle Prozesse und Abläufe effizient und zuverlässig ablaufen müssen. Da Verzögerungen in Zwischenschritten sich auf die ganze Auslieferung des Exemplars oder der digitalen Kopie desjenigen, auswirken können.
 
-Im Hochregallager der Speicherbibliothek werden momentan bis zu 110'000 Behältern mit verschiedenen Exemplaren (von welchen viele mit RFID Tags ausgestattet sind) gelagert. Die Behälter werden manuell von Menschen befüllt und anschliessend wird der Behälter voll autonom an einen Lagerplatz gefahren. Zeitweise können auch gewisse Exemplare wieder aus den Behältern entnommen werden um dieses Exemplar zu Lesen, Scannen oder einer der teilnehmenden Bibliotheken zurückzusenden. Während dem Vorgang des Lagerns und Entnehmen der Exemplare werden weiterhin Menschen für das Befüllen und Pflegen der Behälter verwendet. Dies birgt die Gefahr, dass eine Person aus Versehen ein Exemplar in einen falschen Behälter einsortiert. Und so das Exemplar nur sehr umständlich wiedergefunden werden kann \parencite{WickiBaumann2019Projektbeschrieb}.
+Im Hochregallager der Speicherbibliothek werden momentan bis zu 110'000 Behältern mit verschiedenen Exemplaren, von welchen viele mit RFID Tags ausgestattet sind, gelagert. Die Behälter werden manuell von Menschen befüllt und anschliessend voll autonom an einen Lagerplatz gefahren. Zeitweise können auch gewisse Exemplare wieder aus den Behältern entnommen werden um dieses Exemplar zu Lesen, Scannen oder einer der teilnehmenden Bibliotheken zurückzusenden. Während dem Vorgang des Lagerns und Entnehmen der Exemplare werden weiterhin Menschen für das Befüllen und Entleeren der Behälter verwendet. Dies birgt die Gefahr, dass eine Person aus Versehen ein Exemplar in einen falschen Behälter einsortiert. Und so das Exemplar nur sehr umständlich wiedergefunden werden kann \parencite{WickiBaumann2019Projektbeschrieb}.
```

Comment:
Dies birgt die Gefahr, dass eine Person aus Versehen ein Exemplar in einen falschen Behälter **einsortiert. Dies führt zu dem Umstand, dass** das Exemplar nur sehr umständlich wiedergefunden werden kann.

Achtung Satzbeginn mit und ist sehr komisch